src/main/java/kr/pickple/back/crew/service/CrewMemberService.java

```diff
@@ -52,22 +54,36 @@ public CrewProfileResponse findAllCrewMembers(final Long crewId, final Registrat
     }
 
     @Transactional
-    public void crewMemberStatusUpdate(final Long crewId, final Long memberId,
-            final CrewMemberUpdateStatusRequest crewMemberUpdateStatusRequest) {
+    public void crewMemberStatusUpdate(final Long loggedInMemberId, final Long crewId, final Long memberId,
+                                       final CrewMemberUpdateStatusRequest crewMemberUpdateStatusRequest) {
```

Comment:
```suggestion
    public void crewMemberStatusUpdate(
        final Long loggedInMemberId, 
        final Long crewId, 
        final Long memberId,
        final CrewMemberUpdateStatusRequest crewMemberUpdateStatusRequest
    ) {
```
P1
- 저희 팀 컨벤션에 맞게 이렇게 고쳐보면 어떨까 합니다!